docs: update magento2, pimcore, sulu quickstarts (#6822)
ddev/ddev: docs/content/users/quickstart.md

>     # Change the base-url below to your project's URL

Reviewer: Sorry to request other things here, but let's separate the domain name into an environment variable so they can mostly copy/paste. If we say `export MAGENTO_HOSTNAME= my-magento2-site` and then use `mkdir ${MAGENTO_HOSTNAME} && cd ${MAGENTO_HOSTNAME}` then `ddev magento setup:install with the ${MAGENTO_HOSTNAME}.ddev.site` will work without them having to stop and fiddle with it.

>     # Change the base-url below to your project's URL
>     ddev magento setup:install --base-url="https://my-magento2-site.ddev.site/" \

Reviewer: ```suggestion
    export MAGENTO_HOSTNAME=my-magento2-site
    mkdir ${MAGENTO_HOSTNAME} && cd ${MAGENTO_HOSTNAME}
    ddev magento setup:install --base-url="https://${MAGENTO_HOSTNAME}.ddev.site/" \
```

```bash
mkdir my-sulu-site && cd my-sulu-site

> ddev config --project-type=php --docroot=public --upload-dirs=uploads --database=mysql:8.0 --webimage-extra-packages="xmlstarlet"

Author: Thanks, I didn't know it was abandoned in Debian. (I only knew that it was maintained in Arch.)

ddev start
ddev composer create sulu/skeleton

> # Create your default webspace configuration `config/webspaces/my-sulu-site.xml`
> # from `config/webspaces/website.xml`. The command below will adjust the values
> # for `<name>` and `<key>`, so that they are matching your project:

Reviewer: ```suggestion
# Create your default webspace configuration `config/webspaces/my-sulu-site.xml`
# from `config/webspaces/website.xml`. Alternately, the command below will adjust the values
# for `<name>` and `<key>` to match the project you have set up:
```

# <name>My Sulu Site</name>
# <key>my-sulu-site</key>

> ddev exec 'name="$(echo "${DDEV_PROJECT}" | sed "s/\b\(.\)/\U\1/g" | tr "-" " ")"; key="${DDEV_PROJECT}"; xmlstarlet ed -u "//_:webspace/_:name" -v "${name}" -u "//_:webspace/_:key" -v "${key}" -u "//_:portals/_:portal/_:name" -v "${name}" -u "//_:portals/_:portal/_:key" -v "${key}" config/webspaces/website.xml > config/webspaces/${DDEV_PROJECT}.xml && rm -f config/webspaces/website.xml'

Reviewer: (I think this is OK with the modest update above, saying that they can edit themselves or use this magic command. Still a bit uncomfortable with it.) 

I understand that you're trying to simplify the technique here, but I'm not actually convinced that people will understand or be able to recreate what we're doing here. I may prefer the old technique, which has clear docs. We want people to be able to clearly understand every step, and perhaps with this approach we're not doing that.

Author: I'll replace it with simpler `ddev exec sed ...` approach.
